app/models/grda_warehouse/tasks/identify_duplicates.rb

```diff
@@ -222,8 +238,9 @@ def all_source_clients
     end
 
     # figure out who doesn't yet have an entry in warehouse clients
-    private def load_unprocessed
-      GrdaWarehouse::Hud::Client.source.pluck(:id) - GrdaWarehouse::WarehouseClient.pluck(:source_id)
+    private def unprocessed
+      @unprocessed_source_ids ||= GrdaWarehouse::Hud::Client.source.pluck(:id) - GrdaWarehouse::WarehouseClient.pluck(:source_id)
```

Comment:
how did this get past rubocop?  shouldn't that be `@unprocessed ||=`

Reply:
I think rubo lets it pass because the memoized data isn't the returned data... That said, I see no reason to cache the ids, and not the AR...